platformics/api/core/gql_loaders.py

```diff
@@ -126,13 +126,26 @@ async def load_fn(keys: list[Any]) -> typing.Sequence[Any]:
                 order_by: list = []
                 if relationship.order_by:
                     order_by = [relationship.order_by]
-                query = get_aggregate_db_query(
-                    related_model, CerbosAction.VIEW, self.cerbos_client, self.principal, where, aggregate, remote  # type: ignore
+
+                if selections:
+                    aggregate_selections = [selection for selection in selections if getattr(selection, "name") != "groupBy"]
+                    groupby_selections = [selection for selection in selections if getattr(selection, "name") == "groupBy"]
+                    groupby_selections = groupby_selections[0].selections if groupby_selections else []
+                else:
+                    aggregate_selections = []
+                    groupby_selections = []
+                if not aggregate_selections:
+                    raise Exception("No aggregate functions selected")
```

Comment:
Nit: It would be nice to raise a PlatformicsException here, so the error text will actually make it back to the API response